content/docs/get-started/index.md

```diff
@@ -52,29 +59,15 @@ save(
 )
 ```
 
-Here, we load a well-known
+Here, we loaded a well-known
 [Iris flower dataset](https://archive.ics.uci.edu/ml/datasets/iris) with
-scikit-learn and train a simple classifier. But instead of pickling the model,
-we save it with MLEM (check out the full list of supported
-[ML frameworks](/doc/object-reference/model)).
-
-## Productionization
-
-Now MLEM can do the heavy machinery for us, enabling all these scenarios in a
-couple lines of code:
-
-- **[Apply model](/doc/get-started/applying)** - load model in Python or get
-  prediction in command line.
-- **[Serve model](/doc/get-started/serving)** - create a service from your model
-  for online serving.
-- **[Build model](/doc/get-started/building)** - export model into Python
-  packages, Docker images, etc.
-- **[Deploy model](/doc/get-started/deploying)** - deploy your model to Heroku,
-  Sagemaker, Kubernetes, etc.
+`scikit-learn` and trained a simple classifier. Instead of pickling the model,
+we persisted it using MLEM's `save` API.
```

Comment:
"To persist" doesn't really mean [what we want it to](https://github.com/iterative/dvc.org/issues/2960). "Save" was better anyway (it's simple and it matches the MLEM fn name).